examples/browser-api-playground/src/useAuthProvider.ts

```diff
@@ -40,16 +41,30 @@ const fetchTokenFromServer = async (
   );
 };
 
-const useAuthProvider = (authOptions: AuthOptions, backend?: string) => {
-  const [authState, setAuthState] = useState<AuthState>(defaultAuthState);
+const useAuthProvider = () => {
+  const { config } = useContext(EmbedConfigContext)
+  const authOptions = config[authOptionsKey]
+  const { backend, authMethod}  = config[baseOptionsKey]
+
+  const [authState, setAuthState] = useState<AuthState>(() => {
+    /**
+     * Add onAuthTokenRequired if authMethod = token as QA automation uses window.config 
+     * to set the config value and functions are omitted in JSON.stringify which we do
+     * before storing in session/local storage
+     */
+    if (authMethod === 'token') {
+      defaultAuthState.onAuthTokenRequired = () => {}
```

Comment:
Having a discussion with Nithin on this, if we can alert when token expires so we can put an automation test for onAuthTokenRequired flow as well(will be adding before merging)

Reply:
Don't totally follow - do you mean when the short-lived token expires?

Reply:
One way to simulate the return of a token (can we manipulate / mock the response for the token itself or simply manipulate the generated token), and the expect that onAuthTokenRequired is called by the web SDK.

Reply:
Discussed offline, got confused between global auth token and user level. Will be taking enabling this flow separately(not in this PR). Thanks for clarifying 😄